diff --git a/node_modules/@crxjs/vite-plugin/dist/index.mjs b/node_modules/@crxjs/vite-plugin/dist/index.mjs
index 4e279d6..d684796 100644
--- a/node_modules/@crxjs/vite-plugin/dist/index.mjs
+++ b/node_modules/@crxjs/vite-plugin/dist/index.mjs
@@ -1712,7 +1712,14 @@ function compileFileResources(fileName, {
   assets: /* @__PURE__ */ new Set(),
   css: /* @__PURE__ */ new Set(),
   imports: /* @__PURE__ */ new Set()
-}) {
+},
+processedFiles = new Set(),
+) {
+  if (processedFiles.has(fileName)) {
+    return resources;
+  }
+  processedFiles.add(fileName);
+
   const chunk = chunks.get(fileName);
   if (chunk) {
     const { modules, facadeModuleId, imports, dynamicImports } = chunk;
@@ -1721,7 +1728,7 @@ function compileFileResources(fileName, {
     for (const x of dynamicImports)
       resources.imports.add(x);
     for (const x of [...imports, ...dynamicImports])
-      compileFileResources(x, { chunks, files, config }, resources);
+      compileFileResources(x, { chunks, files, config }, resources, processedFiles);
     for (const m of Object.keys(modules))
       if (m !== facadeModuleId) {
         const key = prefix$1("/", relative(config.root, m.split("?")[0]));
@@ -1734,7 +1741,8 @@ function compileFileResources(fileName, {
             compileFileResources(
               script.fileName,
               { chunks, files, config },
-              resources
+              resources,
+              processedFiles
             );
           }
       }
